ONSdigital/florence
# Add websocket connection between client->server

### What

Add websocket functionality to the Florence server and an API in client-side Florence to send/receive messages over it. 

### How to review

Check that:
1) Example websocket messages on app load are received on the server and acknolwedged on the client.
2) Disable the connection, send a request, enable connection, check that the websocket messages that couldn't be sent whilst the connection were down were sent once the connection was re-established.
3) Check the code looks sensible.

### Who can review

Anyone but me (possibly @ian-kent and @jondewijones).
